metrics.yaml

```diff
@@ -0,0 +1,29 @@
+---
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+usage:
```

Comment:
Fly-by: glean parser can be used to generate the _required_ metric documentation. In other platforms/bindings, this is usually deep integrated within the build system (e.g. Gradle on Android), making it transparent to the user and guaranteeing that it's almost always up to date. While I'm not sure we already have a good story about that on Python, there's still value in manually generating the docs.

In this specific case it might be worthwhile to add a comment at the top of the file explaining how to generate docs.

Reply:
I filed [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1624559) for further discussions.

Reply:
Thanks Alessio! I think this is a little less important for mozregression where we're only collecting a small number of things and it's easy to keep things updated by hand (P.S. some docs in #572 if you want to take a look) but I agree it would be nice to get this solved more generally.

Reply:
My pleasure! I would encourage you to use still generate and commit the generated docs (even for a small number of metrics), since the format is the same across all products and you get important details (who owns the metric? where is the data review?) along the description :)

Reply:
@wlach I think this comment slipped through. I would really strongly encourage you to generate docs with the glean parser. These are the doc-o-types that were verified by other teams to make sure they are consistent, readable and have all the information require to make them useful.

You can do so by manually running:

`glean_parser translate -f markdown -o docs/ metrics.yaml pings.yaml`

Reply:
Done! Thanks for the clarification.